src/Propel/Generator/Model/CrossForeignKeys.php

```diff
@@ -130,9 +130,9 @@ public function getIncomingForeignKey()
      * @param  ForeignKey $fk
      * @return bool
      */
-    public function isAtLeastOneLocalPrimaryKeyNotCovered(ForeignKey $fk)
+    public function isAtLeastOneLocalColumnNotCovered(ForeignKey $fk)
```

Comment:
This change would invalidate the above comment as the logic is expanded beyond PKs.